Microsoft.Toolkit.Services/Microsoft.Toolkit.Services.csproj

```diff
@@ -14,6 +14,7 @@
     <!-- This is a temporary workaround for https://github.com/dotnet/sdk/issues/955 -->
     <DebugType>Full</DebugType>
     <NoWarn>CS8002;CS0618</NoWarn>
+    <DeterministicSourcePaths Condition="'$(EnableSourceLink)' == ''">false</DeterministicSourcePaths>
```

Comment:
Why is this set to false?

Reply:
The build was failing with:
```
C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\Bin\Roslyn\Microsoft.Managed.Core.targets(104,5): error : SourceRoot items must include at least one top-level (not nested) item when DeterministicSourcePaths is true
```
Build here: https://dev.azure.com/dotnet/WindowsCommunityToolkit/_build/results?buildId=33784&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=d1e88c09-dceb-53c1-5579-19c846f3526a
This issue https://github.com/dotnet/sourcelink/issues/91 gave me the hint for this work-around. I've also enabled source link for the test projects, which was also failing for a couple of our test projects https://dev.azure.com/dotnet/WindowsCommunityToolkit/_build/results?buildId=33773&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=d1e88c09-dceb-53c1-5579-19c846f3526a, which is also mentioned on that issue.
Is there a different/better fix?

Reply:
Ah, yes, that's probably https://github.com/NuGet/Home/issues/5894